static/app/icons/icons.stories.tsx

```diff
@@ -1543,11 +1543,25 @@ export default function IconsStories() {
       .map((name): TIcon => ({id: name, name})),
   };
 
-  const allIcons: TIcon[] = SECTIONS.flatMap(section => section.icons);
-
-  const variants = Object.keys(theme.tokens.content);
-  const shuffled = [...allIcons].sort(() => Math.random() - 0.5);
-  const picked = shuffled.slice(0, variants.length);
+  const randomIcons = useMemo(() => {
+    const variants = Object.keys(theme.tokens.content);
+    const allIcons: TIcon[] = SECTIONS.flatMap(section => section.icons);
+    const shuffled = [...allIcons].sort(() => Math.random() - 0.5);
```

Comment:
What if we just used IconSentry for all the examples? Then the only thing changing is the `variant`, not multiple variables at once, across pageloads.

More clarity for the user (me!)